MetaMask/metamask-extension
# fix: BlockaidBannerAlert other/fallback description i18n

## **Description**

Fixes translation bug mentioned by @matthewwalsh0 ([Internal thread](https://consensys.slack.com/archives/C04T0NWJ16E/p1704989891937549))
> This line references an other message that doesn't seem to exist and so causes a full page MetaMask error.

note: 
- users should have **not** experienced the bug. if they did, we would expect to see a related Sentry error
- relevant translation is used to display in case we receive an unrecognized reason from PPOM 

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/22492

## **Manual testing steps**

1. open storybook `yarn storybook`
2. go to http://localhost:6006/?path=/docs/components-app-securityproviderbanneralert-blockaidbanneralert--docs
3. replace "reason" value with an unidentified string value
4. observe description displays as expected and no errors
## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
